content/references.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: 'Reference'
+metaTitle: 'Reference'
+metaDescription: 'Links to helpful reference articles for building on Element.'
 ---
 
-- [Visual Guide to Proptypes](references/proptypes)
+- [Proptypes](references/proptypes)
 - [Block Utils](references/block-utils)
-- [Volusion API ElementSdk Client Reference](references/sdk)
+- [SDK](references/sdk)
```

Comment:
@elisabethnewell This section is specifically for `ElementSdk.client`. Do you still want to call it just `SDK`?

Reply:
The page title is just "SDK," do we want to change that to "Element SDK"?

Reply:
As long as we're not changing the file name, we can change the title. But I'm saying it's not just the SDK. It's specifically the client content within the SDK.

Reply:
Would "Element SDK Client" suffice?

Reply:
Sure.